transifex/native/cds.py

```diff
@@ -240,3 +236,24 @@ def _serialize(self, source_string):
             data['meta']['context'] = source_string.context
 
         return source_string.key, data
+
+    def _get_headers(self, use_secret=False, extra_headers=None):
```

Comment:
Minor: Since we are smart about the `use_secret` argument, couldn't we be as smart with the `etag`? Why not include an `etag=None` keyword argument here and not do the `extra_headers` bit?

Reply:
Sure. I thought of adding something like `use_etag` but there was no way of retrieving that without the language, and I didn't think of passing the `etag` value as an argument 😛